lib/httpclient.rb

```diff
@@ -44,9 +44,12 @@
 #     clnt = HTTPClient.new
 #
 # 2. Accessing resources through HTTP proxy.  You can use environment
-#    variable 'http_proxy' or 'HTTP_PROXY' instead.
+#    variable 'http_proxy' or 'HTTP_PROXY' or 'SOCKS_PROXY' instead.
```

Comment:
setting "HTTP_PROXY" does not appear to work anymore, only works with "http_proxy"